### ISSUE TITLE
Nie działa pobieranie starszych odcinków Świat według Kiepskich do 297

### ISSUE THREAD

**Issue by ghost** [zacny/voddownloader#132]
Starsze odcinki do 297 które są w jakości SD 576p wywalają błąd: "Nie udało się odnaleźć idetyfikatora."
Od 298 odcinki były w 720p i działa bez problemu

Przmus to kiedyś naprawił, cytat: "Starsze materiały miały krótszy numer w adresie, więc funkcja: "idn = x.match(/[\a-z\d]{32}/)[0];" się wysypywała. Przywróciłem alternatywną, starszą (była w skrypcie już kiedyś) metodę pozyskiwania tego hasha w razie wystąpienia problemu."

**Comment by HerMajestyDrMona** [comment]
Tak, ale to było jeszcze za czasów IPLI. Ogólnie "stara wersja" skryptu wyglądała kompletnie inaczej i opierała się na podstawowym Java-skrypcie. Niestety, ja z lenistwa lub po prostu braku motywacji nie mogę połapać się w tej nowej wersji. Kiedyś próbowałem przywrócić przycisk pobierz na Polsatboxie czy tam Playerze i mimo spędzenia 2 godzin nie udało mi się tego poprawnie zrobić. Po tym incydencie kompletnie przestało mi się chcieć próbować. Kolejna sprawa, że od kilku lat niczego z VOD nie pobierałem i przez to motywacja kompletnie padła. Jakoś ciężko robić coś czego samemu się nie używa.
